DEVELOPING_TUTORIALS.md

```diff
@@ -135,76 +137,74 @@ Depending on which lesson format you've chosen, you'll need to create 2-4 files
 
-##### Vue file
 
-Select the appropriate boilerplate Vue file for your lesson from the `tutorials/boilerplates` directory:
+##### Lesson text file (with optional images)
+
 
-- `boilerplate-standard.vue` for a lesson with a coding exercise which does not require a file upload
-- `boilerplate-file-upload.vue` for a lesson with a coding exercise that requires a file upload
-- `boilerplate-multiple-choice.vue` for a lesson with a multiple-choice quiz
-- `boilerplate-no-exercise.vue` for a text-only lesson
 
-Copy that boilerplate into the tutorial directory you created earlier (e.g. `0005`) and rename it to the 2-digit number of the lesson.
+Create a `.md` file using the boilerplate and add the markdown-formatted text of the lesson itself (your educational content). The name of this file should be the 2-digit lesson number.
 
-For example, to create a Vue file for a standard coding exercise as Lesson 01 of Tutorial 0005 (while still in `src/tutorials`):
+For example (for Lesson 01 of Tutorial 0005):
 
 ```sh
-> cp boilerplates/boilerplate-standard.vue 0005/01.vue
+> cp boilerplates/boilerplate.md 0005-tutorial-short-title/01.md
 ```
 
-Replace anything in the boilerplate file that reads "REPLACEME" with the 2-digit lesson number.
+If you want to add images to your markdown file, place them in the `public/tutorial-assets` directory, with the following naming convention:
 
-If your lesson includes a coding exercise, you'll also use this file to set up your default code and validation, as described later in these instructions.
+`T<4-digit-tutorial-id>L<2-digit-lesson-number>-<imageName>`, such as `T0001L05-diagram.svg`.
 
-##### Lesson text file (with optional images)
+Then in your lesson markdown file, you can either add it with regular markdown:
 
-Create a `.md` file alongside your `.vue` and add the markdown-formatted text
-of the lesson itself (your educational content). The name of this file should match the 2-digit lesson number used
-in the corresponding Vue file.
+```
+![Description of the image](tutorial-assets/T0001L01-diagram.svg)
+```
 
-For example (for Lesson 01 of Tutorial 0005):
+...or with regular HTML, if you need to set the image size:
 
-```
-src/tutorials/0005/01.md
+```html
+<img src="tutorial-assets/T0001L01-diagram.svg" width="300px" height="150px" />
 ```
 
-If you want to add images to your markdown file, place them in the `public/tutorial-assets` directory, with the following naming convention (you'll use a similar convention when creating the lesson components in the router):
+##### Javascript file
 
-`T<4-digit-tutorial-id>L<2-digit-lesson-number>-<imageName>`, such as `T0001L05-dag.svg`.
+Select the appropriate boilerplate Javascript file for your lesson from the `src/tutorials/boilerplates` directory:
 
-Then in your lesson markdown file, you can either add it with regular markdown:
+- `boilerplate-standard.js` for a lesson with a coding exercise which does not require a file upload
```

Comment:
Do you think we should rename standard to code to match our new icons and lesson types?

Reply:
👍